### DISCUSSION TITLE
How do I  uninstall cmder

### DISCUSSION THREAD

**Discussion by AlejoDLP** [cmderdev/cmder#2433] [Q&A] [answered]
Hello I was trying to uninstall cmder to do a clear instalation but when I erease the folder the screen put blue showing an error

**Comment by daxgames** [comment, marked as answer]
'Uninstalling' cmder is done by deleting the folder.  Cmder is never really installed.  It is just a folder full of files.

If you have ran `/register` to register the [context menu](https://github.com/cmderdev/cmder/wiki/Context-menu-integration), run `Cmder.exe /unregister` before you delete the folder to remove the registry keys.

If you set up the [`%CMDER_ROOT%` environment variable](https://github.com/cmderdev/cmder/wiki/Setting-up-Environment-Variables) manually, you can delete it too.  Follow the wiki to learn how to access it, then select it and clink Delete.

**Reply by AlejoDLP** [reply]
@daxgames I install it with the zip file and It gives me a error it appears me a screen in blue with a sad face like this :(

**Comment by daxgames** [comment]
I dont know.  All you have to do is exit Cmder and dekete the folder.  There is ni install or uninstall process.  

Only thing I can suggest reboot and try again.

**Comment by mrx23dot** [comment]
Also integration unregister entries.
